app/entry.server.tsx

```diff
@@ -1,21 +1,113 @@
-import type { EntryContext } from "@remix-run/cloudflare";
+import { PassThrough } from "node:stream";
```

Comment:
Copied from the fly.io example https://github.com/remix-run/remix/tree/main/templates/fly